mobile/lib/features/trade/trade_bottom_sheet_confirmation.dart

```diff
@@ -163,6 +163,8 @@ class TradeBottomSheetConfirmation extends StatelessWidget {
     // Fallback to 0 if we can't get the fee or the margin
     Amount total =
         tradeValues.margin != null ? Amount(tradeValues.margin!.sats).add(reserve) : Amount(0);
+    total = total.add(tradeValues.fee ?? Amount.zero());
```

Comment:
🤔 It reads weird when we include this fee in the message:

```
X sats will be locked up in a DLC channel.
```

I think before this change it only included the margin + reserve, which I think made sense because everything else is fees. If we include the order-matching fee, I think we should include all the fees.

In general I don't like the way we present all this information, before or after this change, but this change makes it a bit worse for me.

Reply:
It's just weird that we show a total which does not represent a total 😅.

Reply:
Before this change, it was the total money you own in the channel.

After this change, it is the total money you own in the channel + a fee that belongs to the coordinator. It seems like less of a total to me 🤷